modules/core/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -156,6 +159,40 @@
     // public for tests
     public String blobProviderId;
 
+    /**
+     * @since 11.5
+     */
+    public static HeaderElement getHeaderElement(String contentType) {
```

Comment:
What is the point of exposing the an org.apache.http.HeaderElement object? 
Can't we just have 
String getMimeType(String contentType)
String getCharset(String contentType)

Reply:
For the tests, though we could also do `BasicHeaderValueParser.parseHeaderElement(contentType, null)` in the tests directly...

Reply:
Yes we could inline the code of getHeaderElement directly in the two methods, this is not a time-critical method that's called a lot so we can afford to parse the header twice.